.github/workflows/flaky.yaml

```diff
@@ -87,7 +87,7 @@ jobs:
             echo "$failure" >> $GITHUB_OUTPUT
           done
           echo "" >> $GITHUB_OUTPUT
-          echo "See https://github.com/n0-computer/iroh/actions/workflows/flaky.yaml" >> $GITHUB_OUTPUT
+          echo "See https://github.com/n0-computer/iroh-blobs/actions/workflows/flaky.yaml" >> $GITHUB_OUTPUT
```

Comment:
```suggestion
          echo "See https://github.com/${{ github.repository }}/actions/workflows/flaky.yaml" >> $GITHUB_OUTPUT
```
I'm trying to get them all onto something that doesn't trigger copy-paste errors.  This is how this started...

Reply:
Ah right, I forgot about that.